tests/test_formatter.py

```diff
@@ -1,9 +1,11 @@
 import unittest
 import optparse
 import os
+import sys
 import tempfile
 from flake8_formatter_junit_xml import JUnitXmlFormatter
 from flake8 import style_guide
+from io import StringIO
```

Comment:
```suggestion
try:
    # This is for Python2 only.
    from StringIO import StringIO
except ImportError:
    # This also exists in Python2 but printing to it causes unicode errors.
    from io import StringIO
```

Reply:
I tried this branch locally with python 2.7 and on a first glance, it seemed to work fine with `--tee` and `--output-file`. As far as I can tell, the only reason the tests fail for python 2 is that the tests replace `sys.stdout` with `io.StringIO`.

This script:
```python
from __future__ import print_function
import io
import sys
print('test1')
sys.stdout = io.StringIO()
print('test2')
```
prints:
```
test1
Traceback (most recent call last):
  File "/tmp/a.py", line 9, in <module>
    print('test2')
TypeError: unicode argument expected, got 'str'
```
This problem doesn't occur when `StringIO.StringIO` is used instead of `io.StringIO`.

Reply:
Thank you for suggestion! Your suggestion works well and now ci becomes all green :)